ext/ffi/config.m4

```diff
@@ -10,12 +10,6 @@ if test "$PHP_FFI" != "no"; then
     FFI_INCDIR=$PHP_FFI/include
     FFI_LIBDIR=$PHP_FFI
   else
-    dnl First try to find pkg-config
-    if test -z "$PKG_CONFIG"; then
```

Comment:
Is it okay to drop this without a full migration to PKG_CHECK_MODULES? Do we already initialize PKG_CONFIG somewhere else?

Reply:
`PKG_CONFIG` is initialized via `PKG_PROG_PKG_CONFIG` in configure.ac. `AC_PATH_PROG(PKG_CONFIG, pkg-config)` actually overwrites that path. `PKG_PROG_PKG_CONFIG` should probably be wrapped in an `AC_REQUIRE` macro to actually require `pkg-config` to be installed.

Reply:
Thanks for the clarification, this is fine then.

Reply:
Do you want me to add the `AC_REQUIRE` macro?

Reply:
It would be good to check that $PKG_CONFIG was found, yes. But is AC_REQUIRE the way to do that? I thought AC_REQUIRE is about macro dependencies.

Reply:
I had a closer look at pkg.m4 and found using `AC_REQUIRE` is not necessary, since that's what `PKG_CHECK_MODULES` does internally if `PKG_PROG_PKG_CONFIG` has not already been called.